docs/configuration-format.md

```diff
@@ -0,0 +1,152 @@
+The definition of exporter is specific for each type however at the minimum it will include an endpoint string (which normally includes the destination address and port).
+The structure of exporter configuration is the following:
+
+```yaml
+<exporter name>: # key is string, unique name of exporter
+  endpoint: <network interface and port to bind to, address:port>
+  # other key/value pairs as needed by specific exporter type
+```
+
+The format and interpretation of `endpoint` is exporter specific but typically is a string in the form `address:port`.
+
+## Service
+
+The effective configuration specifies one or more Pipelines. A pipeline defines how data is received (via `receivers`), processed (via `processors`) and exported (via `exporters`).
+
+A pipeline can be one of the 3 types: `traces`, `metrics`, or `logs`. For a pipeline, `receivers` and `exporters` must be present for pipeline to be complete.
+
+At least one complete pipeline must be defined otherwise the configuration is not valid.
+
+Each pipeline has a name that must be unique within the set of pipelines of the same type (note: it is allowed to have same-named pipelines if they are of different types).
+Note that the configuration may specify receivers, processors and exporters which are not used (referenced) in any pipeline. This makes those receivers, processors and exporters inactive.
+
+Additionally, the service configuration allows users to configure extensions to enable as well as the telemetry to be produced by the Collector.
+
+Here is the configuration structure service configuration:
+
+```yaml
+pipelines:  
+  <pipeline name>: # key is string, unique name of pipeline
+    receivers: [receiver-name-1, receiver-name-2, ...]
+    processors: [processor-name-1, processor-name-2, ...]
+    exporters: [exporter-name-1, exporter-name-2, ...]
+# optionally enable extensions
+extensions: [extension-name-1, extension-name-2, ...]
+# optionally configure telemetry for the collector
+telemetry:
+  <signal name>:
+  # other key/value pairs as needed by specific signal
+```
+
+The name should be in the form `type[/name]`, where type is the input type of the pipeline (either traces, metrics, or logs), and name suffix is optionally appended after a forward slash to ensure uniqueness of the full name of the pipeline. It is not allowed to define more than one pipeline with the same name.
+
+## Pipeline Processors
+
+Pipeline `processors` define what processors will be performed on data after it is received. Each element in ordered-processors list references a processor that is defined in the `processors` section. The order in which processors are listed in `processors` is significant.
+
+## Merging Rules
+
+Default and User-defined configurations are merged at startup and form the Effective configuration that defines the runtime behavior. Merging is performed by recursively applying the following rules to each section:
```

Comment:
We don't really have a recursively merged "default" config. We have default per component. I think this text is misleading.